### PR TITLE
[9.0][MIG] sale_partner_incoterm: migrate to 9.0

### PR DESCRIPTION
**Pull request by lbelorgey**
Migrate the module "sale_partner_incoterm" from v8 to v9 (not yet in v9 branch because added after).

### CODE REVIEW

File: sale_partner_incoterm/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (atchuthan) [inline sale_partner_incoterm/__openerp__.py]: Change the comments as per the latest OCA convention
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3

**Review by atchuthan — APPROVED** [review]
Minor changes with regards to convention, otherwise Runbot + code review :+1:

File: sale_partner_incoterm/__openerp__.py (hunk 1 above)
Comment (lbelorgey) [reply sale_partner_incoterm/__openerp__.py]: Done

**Comment by atchuthan** [comment]
@lbelorgey the same convention needs to be applied for all python & xml files :)

**Comment by yvaucher** [comment]
there are already 2 attempt at porting this module:

https://github.com/OCA/sale-workflow/pull/451
https://github.com/OCA/sale-workflow/pull/389

**Comment by rousseldenis** [comment]
@lbelorgey Is this still relevant? As no activity, it will be closed soon.

**Pull request closed without merging**